ui/apps/ui/src/app/components/results-with-pagination/result.component.ts

```diff
@@ -31,25 +31,43 @@ import { IOffer } from '@collections/data/bundles/bundle.model';
         [q]="q$ | async"
         (activeFilter)="setActiveFilter($event.filter, $event.value)"
       ></ess-colored-tags>
-
-      <ess-url-title
-        [title]="title"
-        [highlight]="highlightsreal['title'] ?? []"
-        [url]="
-          type.value === 'bundle'
-            ? redirectService.internalUrl(
-                validUrl,
-                id,
-                type.value,
-                offers[0]?.main_offer_id
-                  ? '#offer-' + offers[0].main_offer_id.toString().substring(2)
-                  : ''
-              )
-            : redirectService.internalUrl(validUrl, id, type.value, '')
-        "
-      >
-      </ess-url-title>
-
+      <div style="display: flex;">
+        <div class="provider-box" *ngIf="type.value === 'provider'">
+          <div class="provider-offer-pic">
+            <img src="assets/provider.svg" />
+          </div>
+        </div>
+        <div style="display: flex; flex-direction: column;">
+          <ess-url-title
+            [ngClass]="{ padding_for_provider: type.value === 'provider' }"
+            [title]="title"
+            [highlight]="highlightsreal['title'] ?? []"
+            [url]="
+              type.value === 'bundle'
+                ? redirectService.internalUrl(
+                    validUrl,
+                    id,
+                    type.value,
+                    offers[0]?.main_offer_id
+                      ? '#offer-' +
+                          offers[0].main_offer_id.toString().substring(2)
+                      : ''
+                  )
+                : redirectService.internalUrl(validUrl, id, type.value, '')
+            "
+          >
+          </ess-url-title>
+          <ess-url-title
+            *ngIf="type.value === 'provider'"
+            [ngClass]="{ sub_title: type.value === 'provider' }"
+            [title]="abbreviation"
+            [highlight]="highlightsreal['abbreviation'] ?? []"
+            [url]="redirectService.internalUrl(validUrl, id, type.value, '')"
+            style="margin-bottom: -10px;"
+          >
+          </ess-url-title>
+        </div>
```

Comment:
Again - I'm not a fan of switch style templates. But this template already had bundles as a special condition, so it's passable. But will need refactoring in the future

Reply:
I used the else if template